anaconda.spec.in

```diff
@@ -168,6 +150,36 @@ Obsoletes: booty <= 0.107-1
 The anaconda-core package contains the program which was used to install your
 system.
 
+%package install-env-deps
+Summary: Installation environment specific dependencies
+Requires: udisks2-iscsi
+Requires: libblockdev-plugins-all >= %{libblockdevver}
+# active directory/freeipa join support
+Requires: realmd
+Requires: isomd5sum >= %{isomd5sum}
+%ifarch %{ix86} x86_64
+Requires: fcoe-utils >= %{fcoeutilsver}
+%endif
+# likely HFS+ resize support
+%ifarch %{ix86} x86_64
+%if ! 0%{?rhel}
+Requires: hfsplus-tools
+%endif
+%endif
+# kexec support
+Requires: kexec-tools
+Requires: createrepo_c
+# run's on TTY1 in install env
+Requires: tmux
+# install time crash handling
+Requires: gdb
+Requires: rsync
+
+%description install-env-deps
+The anaconda-install-env-deps metapackage lists all installation environment dependencies (mostly storage related).
```

Comment:
I would remove that `(mostly storage related)` part. There is for example tmux and also it is not that useful information for user.

Reply:
OK, dropping that.